src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java

```diff
@@ -0,0 +1,395 @@
+package pl.edu.pw.mini.msi.knowledgerepresentation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.ActionDomain;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentenceParts.*;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentences.*;
+import pl.edu.pw.mini.msi.knowledgerepresentation.grammar.ActionLanguageBaseListener;
+import pl.edu.pw.mini.msi.knowledgerepresentation.grammar.ActionLanguageParser;
+
+public class ActionLanguageListener extends ActionLanguageBaseListener {
+
+    private static final Logger log = LoggerFactory.getLogger(ActionLanguageListener.class);
+
+    private Action lastAction = null;
+    private Action preLastAction = null;
+    private Actor lastActor = null;
+    //private ActionLanguageParser.LogicalExpressionContext lastLogExpr = null;
+    private EQueryType lastQueryType = EQueryType.always;
+    private Fluent lastFluent = null;
+    private IFormula lastFormula = null;

```

Comment:
\[Checkstyle\] ERROR: Variable 'lastFormula' explicitly initialized to 'null' \(default value for its type\)\.